include/crypto/bn.h

```diff
@@ -61,7 +61,7 @@ void bn_set_static_words(BIGNUM *a, const BN_ULONG *words, int size);
 int bn_set_words(BIGNUM *a, const BN_ULONG *words, int num_words);
 
 /*
- * Some BIGNUM functions assume most significant limb to be non-zero, which
+ * Some BIGNUM functions assume most significant limb to be nonzero, which
```

Comment:
Shouldn't it be: _assume **the** most significant limb to be_

Reply:
right. the same wording was used as well in crypto/bn/bn_div.c